bin/standalone-installer-unix

```diff
@@ -0,0 +1,175 @@
+#!/bin/bash
+#
+# Bash program to download the latest Nextstrain CLI standalone installation
+# archive for Linux or macOS, extract it into the current user's app data
+# directory, and check if PATH includes the installation destination.
+#
+# It maintains rough parity with the PowerShell program for Windows,
+# standalone-installer-windows.
+#
+# Set DESTINATION to change the installation location.
+#
+# Set VERSION to change the version downloaded and installed, or pass the
+# desired version as the first argument to this program.
+#
+set -euo pipefail
+shopt -s failglob
+
+: "${NEXTSTRAIN_DOT_ORG:=https://nextstrain.org}"
+
+# Globals declared here to make them more obvious, but set by main() to avoid
+# source ordering requirements and delay execution until the full file is
+# parsed.
+declare KERNEL TARGET DESTINATION VERSION
+
+# Wrap everything in a function which we call at the end to avoid execution of
+# a partially-downloaded program.
+main() {
+    KERNEL="${KERNEL:-$(uname -s)}"
+    TARGET="${TARGET:-$(target-triple)}"
+    DESTINATION="${DESTINATION:-$(destination)}"
+
+    VERSION="${1:-${VERSION:-latest}}"
+
+    local archive archive_url tmp
+
+    archive="standalone-${TARGET}.tar.gz"
+    archive_url="${NEXTSTRAIN_DOT_ORG}/cli/download/${VERSION}/${archive}"
+
+    # Move into a temporary working dir
+    tmp="$(mktemp -d)"
+    if ! debug; then
+        trap "rm -rf $tmp" EXIT
+    fi
+    pushd "$tmp" >/dev/null
+    log "Temporary working directory: $tmp"
+
+    log "Downloading $archive_url"
+    curl "$archive_url" \
+        --fail --show-error --location --proto "=https" \
+        > "$archive"
+
+    log "Extracting $archive"
+    mkdir standalone
+    tar xz$(if-debug v)f "$archive" -C standalone
+
+    if [[ -d "$DESTINATION" ]]; then
+        log "Removing existing $DESTINATION"
+        rm -rf "$DESTINATION"
+    fi
+
+    log "Installing to $DESTINATION"
+    mkdir -p "$(dirname "$DESTINATION")"
+    mv standalone "$DESTINATION"
+
+    popd >/dev/null
+
+    if ! debug; then
+        log "Cleaning up"
+        rm -rf "$tmp"
+    fi
+
+    cat <<~~
+______________________________________________________________________________
+
+Nextstrain CLI ($("$DESTINATION"/nextstrain --version)) installed to $DESTINATION.
+
+~~
+
+    if [[ "$(type -p nextstrain 2>/dev/null)" != "$DESTINATION/nextstrain" ]]; then
+        cat <<~~
+To make the "nextstrain" command available without using its full path,
+please add the following to your shell initialization file (e.g. ~/.bashrc or
+~/.zshrc):
+
+    export PATH="$DESTINATION:\$PATH"
+
+and then logout and login again.  You only need to do this once.
+
+Your default shell appears to be $(default-shell).
+~~
+    fi
+}
+
+target-triple() {
+    local machine vendor os
+
+    machine="$(uname -m)"
```

Comment:
```suggestion
    machine="x86_64"
```

Reply:
I'll modify to use `x86_64` unconditionally on `darwin` systems, but check `uname -m` and error if it's not `x86_64` on others.

Reply:
[Done](https://github.com/nextstrain/cli/compare/bcf777a286854b2f5de4827fb7d984ebb2b2204d..e32a20cba15114ccbe8009fd9276079ab1152051).